### ISSUE TITLE
Error trying to install Helm on OpenShift v3.11.157

### ISSUE THREAD

**Issue by joscorga** [helm/helm#7276]
Output of `helm version`:
Client: &version.Version{SemVer:"v2.9.0", GitCommit:"f6025bb9ee7daf9fee0026541c90a6f557a3e0bc", GitTreeState:"clean"}

Output of `kubectl version`:
Client Version: version.Info{Major:"1", Minor:"11+", GitVersion:"v1.11.0+d4cacc0", GitCommit:"d4cacc0", GitTreeState:"clean", BuildDate:"2019-08-02T00:03:59Z", GoVersion:"go1.10.8", Compiler:"gc", Platform:"linux/ppc64le"}
Server Version: version.Info{Major:"1", Minor:"11+", GitVersion:"v1.11.0+d4cacc0", GitCommit:"d4cacc0", GitTreeState:"clean", BuildDate:"2019-12-02T08:31:13Z", GoVersion:"go1.10.8", Compiler:"gc", Platform:"linux/ppc64le"}

Cloud Provider/Platform (AKS, GKE, Minikube etc.): 
OpenShift v3.11.157

Trying to install helm on OpenShift v3.11 following the instructions listed here: https://blog.openshift.com/getting-started-helm-openshift/ after installing the server, the pod is created but it fails and the error in the logs is: 
`standard_init_linux.go:178: exec user process caused "exec format error"`

Hopefully you can help me figure this out. Thanks!

**Comment by bacongobbler** [comment]
looks like you're running on a ppc64le host. The blog post you referenced fetches Helm v2.9.0 with linux/amd64 architecture.

Please make sure to fetch the right architecture for your platform. In your case, you'd want one for ppc64le.

ppc64le: https://get.helm.sh/helm-v2.9.0-linux-ppc64le.tar.gz 
checksum: https://get.helm.sh/helm-v2.9.0-linux-ppc64le.tar.gz.sha256

I'd also recommend using a more recent version of Helm. The latest version for Helm 2 is currently 2.16.1. That blog post was released on Sept 2017, over 2 years ago.

Hope this helps.

**Comment by bacongobbler** [comment]
One other thing: `helm init` does not provide a ppc64le image for tiller. Have a look at the tiller-multiarch project on how you could use that to build a ppc64le-compatible version of Tiller. Right now I don't think there's anyone in the community (asides yourself) using that.

https://github.com/jessestuart/tiller-multiarch

An alternative option would be to use Helm 3, which removed Tiller. That way you can use and interact with Helm without having to worry about your Kubernetes cluster's host environment.

**Comment by joscorga** [comment]
Hi thanks for your response, I'm sorry I'm new to openshift and kubernetes. Where should I set this url https://get.helm.sh/helm-v2.9.0-linux-ppc64le.tar.gz ? I've just followed the blog as I mentioned and trying to understand, should I replace this url inside the `https://github.com/openshift/origin/raw/master/examples/helm/tiller-template.yaml` under the `spec:
        containers:
        - name: tiller
          image: gcr.io/kubernetes-helm/tiller:${HELM_VERSION}
          env:`
line?

Regarding the `helm init` you mean even if I install the server it won't work? I'm running the `./helm init --client-only` locally in my machine which would install the client so that would be enough?

At the end I'm trying to install helm so I can install the DB2 chart provided here:

https://github.com/IBM/Db2/tree/develop/deployment

Thanks for your help!
